packages/apollo-gateway/src/snapshotSerializers/queryPlanSerializer.ts

```diff
@@ -71,7 +71,7 @@ function printNode(
         ) +
         config.spacingOuter +
         indentation +
-        (node.internalFragments.size > 0
+        (node.internalFragments && node.internalFragments.size > 0
```

Comment:
I haven't figured out _why_, but this was failing a number of tests and is the cause of the other unrelated changes to `internalFragments` code in a few places.

Reply:
Reasons unknown, but the issue didn't persist after removing this code so *shrug*